joomla/joomla-cms
# [4.0] Update UCM component table classes

### Summary of Changes

This updates entries in `#__content_types` table to include new fully qualified table class names.
Components that use `JTable` prefixed core classes are currently not included but maybe they should be. Looking for feedback on this.

### Testing Instructions

This will need to be tested both for new installations and for upgrades from 3.10.
Create a contact, a newsfeed, a tag, a banner, a banner client and a user note.
Make some edits and save.
In the form, click `Versions` button and check whether the current version is marked with a star.

### Expected result

Marked.

### Actual result

Not marked.

### Documentation Changes Required

No.
